README.md

```diff
@@ -192,6 +193,60 @@ cp -r config test-app
 (cd test-app && ./backend)
 ```
 
+### Docker
+
+#### Variables
+
+Setting the following environment variables will be helpful for keeping build and run parameters consistent:
+
+```bash
+export NAME=app
+export VERSION=$(git rev-parse HEAD)
+export CONFIGDIR=/path/to/obelisk/config
+```
+
+#### Building
+
+To build a Docker image using a default configuration, run:
+
+```bash
+docker load < $(nix-build -A dockerImage --no-out-link --argstr name $NAME --argstr version $VERSION)
+```
+
+To build with a custom configuration, Obelisk's default configuration can be imported and overriden in a `release.nix` file added to your top-level directory:
+
+```nix
+{ }:
```

Comment:
A function from the empty set seems not useful. Is there something I'm missing with this?

Reply:
Don't think so, not that I could find from #512